bigchaindb/backend/rethinkdb/config.py

```diff
@@ -0,0 +1,48 @@
+"""Database configuration functions."""
+import logging
+from functools import partial
+
+import rethinkdb as r
+
+from bigchaindb.backend import config
+from bigchaindb.backend.exceptions import DatabaseOpFailedError
+from bigchaindb.backend.rethinkdb.connection import RethinkDBConnection
+
+logger = logging.getLogger(__name__)
+
+TABLES = ('bigchain', 'backlog', 'votes')
```

Comment:
I don't mind to just hardocde for now like elsewhere and to address this in a separate issue if you prefer.